couchbase_cluster_admin/cluster.py

```diff
@@ -85,6 +85,25 @@ def enable_services(self):
         if resp.status_code != 200:
             raise Exception(f"Failed to enable services: {resp.text}")
 
+    def get_cluster_nodes(self):
+        """
+        Gets a list of cluster nodes with their details.
+
+        This is used by the `start_logs_collection()` method to provide a list
+        of node names, since the `nodes='*'` wildcard is not working as
+        documented.
+        """
+        url = f"{self.baseurl}/pools/default"
+
+        resp = self.http_request(url)
+        if resp.status_code != 200:
+            raise GetClusterNodesException(resp.text)
+
+        cluster_info = resp.json()
+        nodes = cluster_info.get("nodes", [])
```

Comment:
When would there be no `nodes` attribute? If there is no such attr, is it suitable to return an empty list, as opposed to raising a specific exception? I'm not sure myself.

Reply:
I will instead use the existing `known_nodes`.